composer.json

```diff
@@ -12,8 +12,8 @@
     }
   },
   "require": {
-    "shopware/core": "~6.5.0",
-    "shopware/storefront":  "~6.5.0"
+    "shopware/core": "~6.6.x-dev",
```

Comment:
@rittou as it's not for stable release yet, please fill up the compatibility table in README.md that `master` branch of the plugin supports `6.6.x-dev` so far, and then it will be changed to `0.5 -> 6.6` once the final release is ready.